src/NServiceBus.Core/Configure.cs

```diff
@@ -126,6 +127,23 @@ internal void Initialize()
             }
         }
 
+        void ConfigureBehaviors()
```

Comment:
I had the same feeling, but I think it shouldn't cause any big issues because the change just no longer registers behaviors which aren't used anyway?